### PR TITLE
fix:#524 symbol service format and count()

### PR DESCRIPTION
**Pull request by Yingxue0323**

### CODE REVIEW

File: app/services/symbolService.js [hunk 1]
```diff
@@ -105,7 +105,7 @@ class SymbolService {
     return { newExchangeSymbol };
   }
 
-  async getPrice(exchangeId = 'binance', symbol = 'BTC/USDT', startDate, endDate, interval = '1d', limit = 1, otherCurrency = 'CNY') {
+  async getPrice(startDate, endDate, exchangeId = 'binance', symbol = 'BTC/USDT', interval = '1d', limit = 1, otherCurrency = 'CNY') {
```
Comment (Wongcheukwai) [inline app/services/symbolService.js]: This is incorrect. Changing the line will affect symbolController and priceScheduler

File: app/services/symbolService.js [hunk 2]
```diff
@@ -131,51 +132,52 @@ class SymbolService {
     
     let usdtRate = 1; 
     if (quote !== 'USDT') {
-      const ticker = await exchange.fetchTicker(symbol + '/USDT');
+      const ticker = await exchange.fetchTicker(`${symbol  }/USDT`);
```
Comment (Wongcheukwai) [inline app/services/symbolService.js]: this alternation should be fine, but you need to it should be this ${ symbol }/USDT`).

File: app/services/symbolService.js (hunk 1 above)
Comment (KevinYang-github) [inline app/services/symbolService.js]: Why add startDate, endDate as parameters? I did not find these two parameters used in this function

Comment (Yingxue0323) [reply app/services/symbolService.js]: > This is incorrect. Changing the line will affect symbolController and priceScheduler

Eslint throwed an error of "Default parameters should be last" here. I've rolled it back to your version and left it to you, hope you can fix that later.

File: app/services/symbolService.js (hunk 2 above)
Comment (Yingxue0323) [reply app/services/symbolService.js]: > this alternation should be fine, but you need to it should be this ${ symbol }/USDT`).

done

File: app/services/symbolService.js (hunk 1 above)
Comment (Yingxue0323) [reply app/services/symbolService.js]: > Why add startDate, endDate as parameters? I did not find these two parameters used in this function

This is @Wongcheukwai 's part, I just adjusted the position of them cuz Eslint throwed an error of "Default parameters should be last" here.
I assume it should be used at the logger in processPriceData(), @Wongcheukwai will fix it.

**Review by ucfyao — COMMENTED** [review]
done

**Pull request merged**